server/api/helpers/mailer/resetPasswordTemplates.js

```diff
@@ -0,0 +1,132 @@
+const verifyTemplate = (username, token) => `
```

Comment:
@oyedejipeace You are suppose to adapt this template to fit your task not use the same words. Please edit the words to reflect the action, specifying that the password link expires in 30mins.